src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -658,6 +658,12 @@
           "type": "object"
         },
 
+        "defaultLevel": {
```

Comment:
would be good here to have a reference to a separate type that contained a shared definition of resultLevel